addons/interact_menu/functions/fnc_keyDown.sqf

```diff
@@ -95,4 +95,79 @@ if (GVAR(menuAnimationSpeed) > 0) then {
                        
 ["interactMenuOpened", [_menuType]] call EFUNC(common,localEvent);
 
+GVAR(firstCursorTarget) = objNull;
+GVAR(firstCursorTargetPos) = [0,0,0];
+//If user has option selected, move Main Actions to cursor position on the vehicle they are looking at (only unmounted, non-self, non-zeus)
+if (GVAR(moveMainInteractionToCursor) && {vehicle ACE_player == ACE_player} && {GVAR(openedMenuType) == 0} && {isNull curatorCamera}) then {
+    private _cursorTargets = lineIntersectsSurfaces [(AGLtoASL positionCameraToWorld [0,0,0]), (AGLtoASL positionCameraToWorld [0,0,10]), ACE_player];
+    TRACE_1("Checking for cursor target",_cursorTargets);
+    if (_cursorTargets isEqualTo []) exitWith {};
+    (_cursorTargets select 0) params ["_intsectPosASL", "", "_target"]; //We need variable `_target` set for the actionPosition call
+    if (isNull _target) exitWith {TRACE_1("Ignoring Terrain (null)",_target);};
+
+    //Ignore Man (confusing, and would have trouble finding a free spot)
+    if (_target isKindOf "CAManBase") exitWith {TRACE_1("Ignoring Man",_target);};
+
+    private _interestModelPos = _target worldToModelVisual (ASLtoAGL _intsectPosASL);
+    private _classActions = missionNamespace getVariable [(format [QGVAR(Act_%1), typeOf _target]), []];
```

Comment:
this line needs to be updated becouse of https://github.com/acemod/ACE3/pull/3547
should be `private _classActions = GVAR(ActNamespace) getVariable typeOf _target;`